test/e2e/app-websocket.ts

```diff
@@ -290,6 +293,95 @@ test(
   })
 );
 
+test(
+  "can grant and revoke zome call capabilities",
+  withConductor(ADMIN_PORT, async (t) => {
+    const {
+      admin,
+      client: appWs,
+      cell_id: cellId,
+    } = await createAppWsAndInstallApp(ADMIN_PORT);
+
+    await admin.authorizeSigningCredentials(cellId);
+    const info = await appWs.appInfo();
+
+    // grant capability
+    const grantRequest: GrantZomeCallCapabilityRequest = {
+      cell_id: cellId,
+      cap_grant: {
+        tag: "test-grant",
+        access: {
+          type: "unrestricted",
+        },
+        functions: {
+          type: "all",
+        },
+      },
+    };
+
+    const grantedActionHash = await admin.grantZomeCallCapability(grantRequest);
+
+    // list capability grants
+    const listRequest: ListCapabilityGrantsRequest = {
+      installed_app_id: info.installed_app_id,
+      include_revoked: true,
+    };
+    const capabilityGrants = await admin.listCapabilityGrants(listRequest);
+    // should have 1 item
+    t.equal(
```

Comment:
Interesting that this library takes actual, expected rather then expected, actual. You've got it the right way round but I hadn't realized that's how the library does it. I should probably double check some of the tests I've written in here :D